src/client/app/components/about.ts

```diff
@@ -50,28 +63,75 @@ export class About implements AfterViewInit {
     currentChosenLanguage = '';
     originalLanguage = '';
 
+    configs = [];
+    selectedConfig = '';
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) {}
+
     ngAfterViewInit() {
         this.availableLanguages = getAvailableLanguages();
         this.currentChosenLanguage = getDisplayLanguage();
         this.originalLanguage = this.currentChosenLanguage;
+        this.configs.push(
```

Comment:
It would be better to reduce coupling and populate this by iterating through environment configs.